src/mame/misc/multfish.cpp

```diff
@@ -424,256 +419,247 @@ A10 <-> A18
 A12 <-> A13
 */
 
-	uint32_t i,j,jscr,romoffset;
-	uint8_t *igrosoft_gamble_gfx = memregion("gfx")->base();
-	std::vector<uint8_t> temprom(igrosoft_gamble_ROM_SIZE);
+	uint32_t jscr,romoffset;
+	uint8_t *gfx = memregion("gfx")->base();
+	std::vector<uint8_t> temprom(ROM_SIZE);
```

Comment:
Is `ROM_SIZE` the actually the size of the ROM region?  If it is, you should just get the size of the ROM region directly.  If it’s potentially less than the size of the ROM region, you should add a check to ensure the ROM region is at least that big to avoid potentially writing of the end of the region.

Reply:
It's size of each gfx ROMs (0x80000 bytes * 8 ROMs).